### PR TITLE
chore: Electron 9 blog post

### PR DESCRIPTION
**Pull request by sofianguy**
cc @VerteDinde (we'll chat more in detail tomorrow 🙂)

Feel free to add a comment for items you think we should highlight in the blog post!

⚠️ do not merge until the following are completed ⚠️
- [x] update chromium version from final chrome roll under *Stack Changes* section
- [x] add a couple new features in tag line sentence
- [x] edit link for M83 blog post
- [x] add a few bullets for *Highlight Features* section
- [x] add any missing items in *Breaking Changes* section
- [x] add items for *API Changes* section
- [x] add items for *Deprecated APIs* section
- [x] add note about [this upcoming change](https://github.com/electron/electron/issues/23506) in *What's Next* section

### CODE REVIEW

File: data/blog/electron-9-0.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* Node.js `12.14.1`
+    * [Node 12.14.1 blog post](https://nodejs.org/en/blog/release/v12.14.1/)
+* V8 `8.3`
+    * [V8 8.1 blog post](https://v8.dev/blog/v8-release-81)
+    * [V8 8.3 blog post](https://v8.dev/blog/v8-release-83)
+
+### Highlight Features
+
+* Multiple improvements to the spellchecker feature. See more details in [#22128](https://github.com/electron/electron/pull/22128) and [#22368](https://github.com/electron/electron/pull/22368).
+* Improved window events handler efficiency on Linux.
```
Comment (sofianguy) [inline data/blog/electron-9-0.md:32]: We should add a commit link to this item. I think it's this one

> * Improved window events handler efficiency on Linux. [#23260](https://github.com/electron/electron/pull/23260)

File: data/blog/electron-9-0.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Changing this default improves the default security of Electron apps, so that apps will need to deliberately opt in to the insecure behaviour. Electron will depreciate the current default of `contextIsolation` in Electron 10.0, and change to the new default (`true`) in Electron 12.0.
+
+For more information on `contextIsolation`, how to enable it easily and it's security benefits please see our dedicated [Context Isolation Document](https://github.com/electron/electron/blob/master/docs/tutorial/context-isolation.md).
+
+### Remove the `nodeIntegration` flag (Starting in Electron 10)
+
+Historically, Electron has recommended that apps use `nodeIntegration: false` to prevent renderers from having access to Electron internals. Over time, it has become clear that this flag actually has negligible security impact and can easily be bypassed. This was the original motivation for adding the contextIsolation flag.
+
+We are now confident enough in the `contextIsolation` feature that we intend to remove the `nodeIntegration` flag and strongly recommend usage of `contextIsolation`. This flag will be deprecated the flag in Electron 10.0, and removed entirely in Electron 12.0.
```
Comment (sofianguy) [inline data/blog/electron-9-0.md:92]:
```suggestion
We are now confident enough in the `contextIsolation` feature that we intend to remove the `nodeIntegration` flag and strongly recommend usage of `contextIsolation`. This flag will be deprecated in Electron 10.0, and removed entirely in Electron 12.0.
```

File: data/blog/electron-9-0.md (hunk 1 above)
Comment (sofianguy) [reply data/blog/electron-9-0.md:32]: and maybe also this
> * Made setting window icons slightly faster on Linux. [#22736](https://github.com/electron/electron/pull/22736)

**Comment by sofianguy** [comment]
pdf viewer https://github.com/electron/electron/pull/22131

File: data/blog/electron-9-0.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+### Change the default of `contextIsolation` from `false` to `true` (Starting in Electron 10)
+
+Without contextIsolation, any code running in a renderer process can quite easily reach into Electron internals or an app's preload script. That code can then perform privileged actions that Electron wants to keep restricted.
+
+Changing this default improves the default security of Electron apps, so that apps will need to deliberately opt in to the insecure behaviour. Electron will depreciate the current default of `contextIsolation` in Electron 10.0, and change to the new default (`true`) in Electron 12.0.
+
+For more information on `contextIsolation`, how to enable it easily and it's security benefits please see our dedicated [Context Isolation Document](https://github.com/electron/electron/blob/master/docs/tutorial/context-isolation.md).
+
+### Remove the `nodeIntegration` flag (Starting in Electron 10)
```
Comment (MarshallOfSound) [inline data/blog/electron-9-0.md:88]: This part is still a but hand wavy and I don't want to lock in version numbers in this blog post.  Can we not mention this removal here.

Comment (sofianguy) [reply data/blog/electron-9-0.md:88]: @MarshallOfSound You mean remove this section completely? Or not mention past Electron 10?

Comment (sofianguy) [reply data/blog/electron-9-0.md]: removed the section

**Pull request merged**